src/librustc_mir_build/hair/cx/mod.rs

```diff
@@ -148,6 +148,7 @@ impl<'a, 'tcx> Cx<'a, 'tcx> {
                 // create a dummy value and continue compiling
                 Const::from_bits(self.tcx, 0, self.param_env.and(ty))
```

Comment:
@oli-obk This isn't new in this PR but I wonder why this isn't undef or something (I guess that might cause ICEs?). Maybe we need a `ty::ConstKind::Err` (if we don't have it already)?

Reply:
We don't have such an `::Err` variant, https://doc.rust-lang.org/nightly/nightly-rustc/rustc/ty/enum.ConstKind.html.

Reply:
Adding one is probably a good idea (but can be done in a separate PR).